samples/Blazor/BlazorWasm/BlazorWasm.Server/Grains/TodoGrain.cs

```diff
@@ -44,8 +44,8 @@ await GrainFactory.GetGrain<ITodoManagerGrain>(item.OwnerKey)
             GrainType, GrainKey, item);
 
         // notify listeners - best effort only
-        GetStreamProvider("SMS")
-            .GetStream<TodoNotification>(item.OwnerKey, nameof(ITodoGrain))
+        this.GetStreamProvider("MemoryStreams")
+            .GetStream<TodoNotification>(StreamId.Create(nameof(ITodoGrain), item.OwnerKey))
```

Comment:
Should we add a 2-parameter overload of `GetStream` accepting `(string ns, string key)`?

Reply:
I think it's a good idea